### PR TITLE
Remove duplicated preamble definition.

### PR DESCRIPTION
**Pull request by takagi**
Merge #326 first.

This PR removes a duplicated preamble definition to use the one introduced in #326.

### CODE REVIEW

File: cupy/sparse/util.py [hunk 1]
```diff
@@ -1,6 +1,24 @@
 import cupy
 
 
+_preamble_atomic_add = '''
+#if __CUDA_ARCH__ < 600
+__device__ double atomicAdd(double* address, double val) {
+    unsigned long long int* address_as_ull =
```
Comment (okuta) [inline cupy/sparse/util.py]: How about using `unsigned long long`?

File: cupy/sparse/compressed.py [hunk 2]
```diff
@@ -201,26 +201,10 @@ def _get_single(self, major, minor):
...
-        assumed = old;
-        old = atomicCAS(address_as_ull, assumed,
-                        __double_as_longlong(val +
-                        __longlong_as_double(assumed)));
-    } while (assumed != old);
-    return __longlong_as_double(old);
-}
-#endif
-'''
         kern = cupy.ElementwiseKernel(
```
Comment (okuta) [inline cupy/sparse/compressed.py]: Please create the kernel outside the method.

File: cupy/sparse/util.py (hunk 1 above)
Comment (takagi) [inline cupy/sparse/util.py]: Fixed!

File: cupy/sparse/compressed.py (hunk 2 above)
Comment (takagi) [inline cupy/sparse/compressed.py]: Fixed!

**Comment by okuta** [comment]
Please check error.
```
ERROR: Failure: AttributeError ('module' object has no attribute 'ElementwiseKernel')
```

**Comment by takagi** [comment]
jenkins test this please

**Comment by okuta** [comment]
LGTM!

**Pull request merged**